src/Overview/components/ProductInfo.jsx

```diff
@@ -9,7 +9,10 @@ const ProductInfo = ({product}) => {
   return (
     <div className='overview-widget-productInfo'>
       <div>
-       <div className='stars-reviews'><StarRating/> <p className='review'>See all reviews</p></div>
+       <div className='stars-reviews'>
```

Comment:
I'll need to make some changes after this pr relating to prop names but good thing I'm reviewing this one. Nice job on these stars btw